tests/tests_test_workflow/test_integ_workflow/integ_test/test_integ_test_runner_opensearch_dashboards.py

```diff
@@ -149,8 +151,6 @@ def test_with_integ_test_ci_groups(self, mock_temp: Mock, mock_test_recorder: Mo
         results = runner.run()
 
         self.assertEqual(results["sql-ci-group-1"], mock_test_results)
-        self.assertEqual(results["sql-ci-group-2"], mock_test_results)
```

Comment:
Can we check his assertions for given ci-group number?

Reply:
This test is checking the assertion when the `ci-group` parameter is passed with value 1. 
I just deleted the previous implementation of running all ci-groups in the manifest file.